src/docs/docs/api.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Making an API Public"
+---
+
+## Introduction
+
+In this document, we explain both how you make an API public and the considerations to take into account before doing so.
+
+## Definitions
+
+### Managed Client
+
+A managed client is one that Sentry controls, such as our frontend. In this type of client, we control both which endpoints are used and the attributes used by those endpoints.
+
+### Partially Managed Client
+
+A partially-managed client is one that we only partially control, since we do not control the version of the client that is running for our users; for example, our Sentry CLI. While we control the endpoints the CLI uses, once the CLI is installed and running in a customer's implementation, we don't control the version of the client a user runs.
+
+### Wild Client
+
+A wild client is a client that Sentry does not control; for example, a script written by a customer run in their repository, or an integration written using the Integration Platform. For this type of client, we have no control over the endpoints they choose to use, nor can we require that they immediately update their endpoint.
+
+### Private API
+
+A private API is an undocumented API. Sentry can make breaking changes to this type of API, and no one will blink an eye.
+
+### Public API
+
+Our public API is documented at [https://docs.sentry.io/api/](https://docs.sentry.io/api/).
+
+### Private Attributes in a Public Endpoint
+
+You can have private attributes within a public endpoint.
+
+As an example: [https://docs.sentry.io/api/teams/retrieve-a-team/](https://docs.sentry.io/api/teams/retrieve-a-team/). The response has multiple attributes:
+
+```json
+{
+  "id": "2",
+  "slug": "the-interstellar-jurisdiction"
+  ...
+}
+```
+
+Let's say we also return a `nickname` for a team. If what is returned is not documented as a response in our public documentation, that attribute is a private attribute within a public endpoint.
+
+## API Versioning
+
+Currently, Sentry's API is v0 and is considered to be in draft phase. While we don't expect our public endpoints to change greatly, keep in mind that our API is still under development.
+
+## Making an endpoint public
+
+When you make an endpoint public, you're committing to always returning at least the set of defined attributes when that endpoint is called. As a result, the attribute(s) **cannot** **be removed** from an endpoint after they are public. Additionally, the type of the attribute **cannot** **be changed** once the attribute is public. The reason the attribute and its type cannot be changed is because both in the wild and managed clients are relying on the public attribute to return what is expected from that endpoint.
```

Comment:
```suggestion
When you make an endpoint public, you're committing to always returning at least the set of defined attributes when that endpoint is called. As a result, the attribute(s) **cannot be removed** from an endpoint after they are public. Additionally, the type of the attribute **cannot be changed** once the attribute is public. The reason the attribute and its type cannot be changed is because both in the wild and managed clients are relying on the public attribute to return what is expected from that endpoint.
```